tests/test_utils.py

```diff
@@ -23,22 +23,17 @@ def test_simple(self):
         d["c"] = 3
         d["a"]
         d["d"] = 4
-        assert len(d) == 3
-        assert "a" in d and "c" in d and "d" in d and "b" not in d
+        assert d.keys() == ["d", "a", "c"]
 
-    def test_itervalues(self):
+    def test_values(self):
```

Comment:
these tests don't seem to be testing iterators? just the `.values()` method

Reply:
The `itervalues` name is left over from when we supported Python 2.